app/models/post.rb

```diff
@@ -103,8 +103,7 @@ def notify
     # Reply from user back to the system
     elsif self.kind == "reply" && self.user_id != self.topic.user_id && self.topic.private?
       I18n.with_locale(self.email_locale) do
-        # NOTE New ticket is misnamed, it should be new-reply
-        PostMailer.new_post(self).deliver_later
+        PostMailer.new_post(id).deliver_later
```

Comment:
Yeah, passing an instance of `ActiveRecord::Base` was deprecated a while ago (rails/rails@23ffd03ede9b27a6cbc3154aa99c247b718ccdbc).

I noticed travis [throwing these deprecation warnings](https://travis-ci.org/helpyio/helpy/builds/176454552) after #436 was merged.

I could probably actually use `self.id` to keep it consistent with the rest of the file. Although the `self` is redundant.